napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -126,8 +126,11 @@ def stop_annotating(self):
         self.record_annotations(self._annotation_model.get_curr_img_index())
 
         # Save rest of annotations, even if empty
-        for idx in range(self._annotation_model.get_curr_img_index()+1, self._annotation_model.get_num_images()):
-            self._annotation_model.add_annotation(self._annotation_model.get_image_at(idx), [])
+        self.record_annotations(self._annotation_model.get_curr_img_index())
```

Comment:
this is duplicate of 126

Reply:
duplicate is removed in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/63ce801a64761c83007581016ab8d270e9b887af